kuksa_databroker/doc/timestamp.md

```diff
@@ -0,0 +1,5 @@
+# Databroker Timestamps
+The implementation of KUKSA databroker shall represent the latest value of a ```Datapoint```. Therefore it adds a ```timestamp``` to a ```Datapoint```. If an actuator/provider/application has no system time the databroker sets the ```timestamp``` to the current system time. For some use cases it can be interesting to provide a timestamp set by the actuator/provider/application. For this we added a so called source timestamp (short ```source_ts```) to a ```Datapoint```. This source timestamp is optional and per default set to None.
```

Comment:
"If an actuator/provider/application has no system time the databroker sets the ```timestamp``` to the current system time"

Doesn't Databroker always set `timetamp`? (Regardless of whether Datapoint contains `source_ts` or not). And strictly speaking, Databroker does not know if actuator/provider/application has system time, it only sees what comes in as timestamp/source-timestamp.

Reply:
Yes